crt_portal/static/js/activity_stream.js

```diff
@@ -0,0 +1,19 @@
+(function(root, dom) {
+  const tagComments = function() {
+    document.querySelectorAll('.activity-stream-item').forEach(item => {
+      const kind = item.innerText.includes('Added comment:') ? 'comment' : 'system';
```

Comment:
This distinction is kind of confusing to me because aren't all the actions technically user actions? i.e. if I send a response letter I would expect that to be a user, not system action. It seems like we should call this "comment only view" or something